### PR TITLE
Add sogrep

### PR DESCRIPTION
**Pull request by svenstaro**
This is from Eli's dotfiles after he'd cleaned it up but never actually went ahead and made this PR.
I figure it's time to add it.

### CODE REVIEW

File: sogrep.in [hunk 1]
```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
```
Comment (jelly) [inline sogrep.in]: Missing license header

**Comment by jelly** [comment]
I don't think this has to be an .in file, is anything converted by the m4 step?

**Comment by anthraxx** [comment]
@jelly please do so if you have some spare time. I don't think we convert anything, its a plain self contained script

**Comment by svenstaro** [comment]
Ok all done.

File: doc/footer.asciidoc [hunk 2]
```diff
@@ -20,7 +20,7 @@ Maintainers:
 * Jan Alexander Steffens (heftig) <jan.steffens@gmail.com>
 * Levente Polyak <anthraxx@archlinux.org>
 * Pierre Schmitz <pierre@archlinux.de>
-* Sébastien Luttringer <seblu@seblu.net> 
+* Sébastien Luttringer <seblu@seblu.net>
```
Comment (svenstaro) [inline doc/footer.asciidoc]: Please just accept this :P

**Comment by svenstaro** [comment]
Can't merge this myself. Can you make contrib?

**Pull request merged**